playground/mocks/data/idp/idx/authenticator-enroll-password.json

```diff
@@ -20,7 +20,7 @@
               "value": [
                 {
                   "name": "passcode",
-                  "label": "Password",
+                  "label": "Enter password",
```

Comment:
Opened up a ticket to fix this - https://oktainc.atlassian.net/browse/OKTA-305923